src/editors/data/services/cms/api.js

```diff
@@ -51,6 +51,33 @@ export const apiMethods = {
       data,
     );
   },
+  checkTranscripts: ({
```

Comment:
"check" doesn't seem super descriptive as to what is going on here. is there any information other than the youtube transcript check which transpires at this endpoint?

Reply:
This API is checking if the video has any transcripts available for import.